configs/sites/hera/packages.yaml

```diff
@@ -20,11 +20,11 @@ packages:
       prefix: /apps/oneapi
   openmpi:
     externals:
-    - spec: openmpi@3.1.4%gcc@9.2.0~cuda+cxx+cxx_exceptions~java~memchecker+pmi+static~wrapper-rpath fabrics=mxm schedulers=slurm
-      prefix: /apps/openmpi/3.1.4/gnu/gcc-9.2.0
+    - spec: openmpi@4.1.5~cuda~cxx~cxx_exceptions~java+lustre~memchecker+pmi~static~wrapper-rpath schedulers=slurm
+      prefix: /scratch1/NCEPDEV/jcsda/jedipara/spack-stack/openmpi-4.1.5
```

Comment:
Note. Installed this in the same place as all other external packages for consistency.